docs/OpenSSLSupport.md

```diff
@@ -173,6 +178,38 @@ X509_load_cert_file | x509_vfy.h | The API loads certificates from the untrusted
 X509_load_crl_file | x509_vfy.h | The API loads CRL from the untrusted host filesystem and adds the CRL to the `X509_STORE` via the X509_STORE_add_crl API. The API is used internally by X509_LOOKUP_hash_dir and X509_LOOKUP_file methods. | The recommendation is not to use this API. An alternative is obtaining in-memory certificates in a secure manner (e.g., secure channel, encrypted storage) and adding the CRL to the `X509_STORE` via X509_STORE_add_crl. |
 X509_load_cert_crl_file | x509_vfy.h | The API is the combination of X509_load_cert_file and X509_load_crl_file. | The recommendation is not to use this API. An alternative is obtaining in-memory certificates in a secure manner (e.g., secure channel, encrypted storage) and adding the certificates/CRL to the `X509_STORE` via X509_STORE_add_cert/X509_STORE_add_crl. |
 
+#### OpenSSL TLS/SSL Configuration
+
+The default set of cipher suites and elliptic curves provided by OpenSSL may include less secure ones.
+To help reducing the risk, OE SDK recommends users to config a TLS/SSL server to use the following cipher suites
+(with the exact order) and elliptic curves.
+
+- TLS 1.3 cipher suites
+  ```
+  TLS13-AES-256-GCM-SHA384
+  TLS13-AES-128-GCM-SHA256
+  ```
+- TLS 1.2 cipher suites:
+  ```
+  ECDHE-ECDSA-AES128-GCM-SHA256
+  ECDHE-ECDSA-AES256-GCM-SHA384
+  ECDHE-RSA-"AES128-GCM-SHA256
+  ECDHE-RSA-AES256-GCM-SHA384
+  ECDHE-ECDSA-AES128-SHA256
+  ECDHE-ECDSA-AES256-SHA384
+  ECDHE-RSA-AES128-SHA256
+  ECDHE-RSA-AES256-SHA384
+  ```
+- Elliptic curve algorithms
+  ```
+  P-521
+  P-384
+  P-256
+  ```
+
```

Comment:
Should we also mention about TLS protocol versions. ?https://liquid.microsoft.com/Web/Object/Read/ms.security/Requirements/Microsoft.Security.Cryptography.10031#TLS%20Protocol%20Versions

Reply:
Update the text.